### PR TITLE
Add stackeddag

### PR DESCRIPTION
**Pull request by davnn**
Checklist
- [x ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x ] Source is from official source.
- [x ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x ] If static libraries are linked in, the license of the static library is packaged.
- [x ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x ] Build number is 0.
- [x ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by davnn** [comment]
@junjihashimoto please confirm the PR, thx!

**Comment by junjihashimoto** [comment]
LGTM! Thank you, too!

File: recipes/stackeddag/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/stackeddag-{{ version }}.tar.gz
+  sha256: 89404050364710f25188812248735fed34dae90c001e35e74bff7fc15a9f970b
+
+build:
```
Comment (synapticarbors) [inline recipes/stackeddag/meta.yaml:12]: This package looks like it should be `noarch: python`. See:
https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds

Comment (davnn) [reply recipes/stackeddag/meta.yaml:12]: I did update the PR. Does this mean [grayskull](https://github.com/conda-incubator/grayskull) failed to infer that the package should be ``noarch: python``?

**Review by xhochy — APPROVED** [review]

**Pull request merged**